benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -886,16 +896,16 @@ void GraphicsBenchmarkApp::UpdateGUI()
     if (IsXrEnabled()) {
         ImVec2 lastImGuiWindowSize = ImGui::GetWindowSize();
         // For XR, force the diagnostic window to the center with automatic sizing for legibility and since control is limited.
-        ImGui::SetNextWindowPos({(GetUIWidth() - lastImGuiWindowSize.x) / 2, (GetUIHeight() - lastImGuiWindowSize.y) / 2}, 0, {0.0f, 0.0f});
-        ImGui::SetNextWindowSize({0, 0});
+        ImGui::SetNextWindowPos({(GetUIWidth() - lastImGuiWindowSize.x) / 2, (GetUIHeight() - lastImGuiWindowSize.y) / 2}, ImGuiCond_FirstUseEver, {0.0f, 0.0f});
+        ImGui::SetNextWindowSize({512, 512}, ImGuiCond_FirstUseEver);
     }
 #endif
 
     // GUI
     ImGui::Begin("Debug Window");
-    GetKnobManager().DrawAllKnobs(true);
-    ImGui::Separator();
     DrawExtraInfo();
```

Comment:
Nit: In `DrawExtraInfo()` there's fields 
(Write Data, Write Bandwidth) that are displayed and which are 0 when quads are not enabled. It's a bit confusing, I think they should not be visible unless quads are enabled or they should be named in a way that links them to the quads?

Reply:
fixed